### DISCUSSION TITLE
Cannot pick the right interface while editing an IP Address

### DISCUSSION THREAD

**Discussion by rkiss** [netbox-community/netbox#15187] [Help Wanted!] [answered]
### Deployment Type

Self-hosted

### NetBox Version

v3.7.2

### Python Version

3.8

### Steps to Reproduce

Create an several devices and each device have an interface with the same name, eg: `eth0`
Create an IP address.
The `Interface` field on the `Interface Assignment` group shows only the interface name and is impossible to tell which interface belong to which device.

### Expected Behavior

The list shall display also the device name, eg: `device1 > eth0` instead of just `eth0`. Also it should be possible to search for `device1`

### Observed Behavior

Only the name of the interface is used, cannot differentiate them.

**Comment by abhi1693** [comment, marked as answer]
Click on the search next to the field on the right

![image](https://github.com/netbox-community/netbox/assets/5083532/93115127-ee7c-4d21-95f8-cb61ae7452a2)

**Comment by rkiss** [comment]
Thank you for your response. Yes, you are correct, you can do it like that. Still, I think it could be better...
Netbox is quite powerful and you can do a lot of things with it. But, maybe because of that, for some simple things you need to do a bit to many clicking and typing.

**Reply by regalialong** [reply]
I honestly agree with the proposal though, it would be nice to have the device / virtual machine name attached to the list, even if we have access to the filters.
